x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/activity_log_date_range_picker/index.tsx

```diff
@@ -29,95 +41,102 @@ const StickyFlexItem = styled(EuiFlexItem)`
 
 export const DateRangePicker = memo(() => {
   const dispatch = useDispatch();
-  const { page, pageSize, startDate, endDate, isInvalidDateRange } = useEndpointSelector(
+  const { page, pageSize, startDate, endDate, autoRefreshOptions } = useEndpointSelector(
     getActivityLogDataPaging
   );
 
-  const onClear = useCallback(
-    ({ clearStart = false, clearEnd = false }: { clearStart?: boolean; clearEnd?: boolean }) => {
-      dispatch({
-        type: 'endpointDetailsActivityLogUpdatePaging',
-        payload: {
-          disabled: false,
-          page,
-          pageSize,
-          startDate: clearStart ? undefined : startDate,
-          endDate: clearEnd ? undefined : endDate,
-        },
-      });
-    },
-    [dispatch, endDate, startDate, page, pageSize]
-  );
+  const [recentlyUsedRanges, setRecentlyUsedRanges] = useState<EuiSuperDatePickerRecentRange[]>([]);
```

Comment:
So the values are only cached until the component is unmounted, right? is that the behaviour in other places in security solution?

Reply:
Yeah, they are reset when component unmounts. Not sure if they are persisted elsewhere in pickers in security_solution, but it's a good idea to do it here anyway. 👍

Reply:
done b638c2df443657be3983ee9780af319235bde2ea and 28a859ab3a5fbbd9a14f4009ded671b969f2dc09